e2e/address-detail/address-detail.po.ts

```diff
@@ -8,7 +8,7 @@ export class AddressDetailPage {
 
   getAddressForSmallScreens() {
     return element(by.css('.element-details > div:nth-of-type(1) > span:nth-of-type(2)'))
-      .getAttribute('textContent');
+      .getAttribute('textContent').then(address => address.trim());
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
      .getAttribute("textContent").then(address => address.trim());
```